sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/SemConvSpanProcessorMetrics.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.trace.export;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.sdk.internal.ComponentId;
+import io.opentelemetry.sdk.internal.SemConvAttributes;
+import javax.annotation.Nullable;
+
+/**
+ * SDK metrics exported for span processors as defined in the <a
+ * href="https://opentelemetry.io/docs/specs/semconv/otel/sdk-metrics/#span-metrics">semantic
+ * conventions</a>.
+ */
+final class SemConvSpanProcessorMetrics implements SpanProcessorMetrics {
```

Comment:
While the logs version will be almost identical, I think it's not worth trying to abstract something since there is a goal to have less internal shared code, and it's still not that much, with the word span showing up in descriptions in a way that it would get pretty awkward anyways

Reply:
I would like less code duplication - but I agree that this repo tolerates a little duplication to make the code easier to read.